bukuserver/templates/bukuserver/bookmarks_list.html

```diff
@@ -8,5 +8,6 @@
 
 {% block tail %}
   {{ super() }}
+  {{ buku.focus(None) }}
```

Comment:
After loading the page, the user will likely want to browse the search results (currently if any filters are supplied the keyboard will be focused on them instead, making keyboard navigation inconvenient).